packages/ui/react-ui-data/src/components/ViewEditor/ViewEditor.tsx

```diff
@@ -102,22 +102,23 @@ export const ViewEditor = ({
       )}
 
       {!readonly && !field && (
-        <div className='flex justify-center p-2'>
-          <div className='flex gap-2'>
-            <Tooltip.Root>
-              <Tooltip.Trigger asChild>
-                <Button onClick={handleAdd} disabled={view.fields.length >= VIEW_FIELD_LIMIT}>
-                  <Icon icon='ph--plus--regular' />
-                </Button>
-              </Tooltip.Trigger>
-              <Tooltip.Portal>
-                <Tooltip.Content classNames='z-[12]'>
-                  {view.fields.length >= VIEW_FIELD_LIMIT ? t('field limit reached') : t('add field')} <Tooltip.Arrow />
-                </Tooltip.Content>
-              </Tooltip.Portal>
-            </Tooltip.Root>
-          </div>
-        </div>
+        <ButtonGroup classNames='p-2 justify-center'>
+          <Tooltip.Root>
+            <Tooltip.Trigger asChild>
+              <IconButton
+                icon='ph--plus--regular'
+                label={t('button add property')}
+                onClick={handleAdd}
+                disabled={view.fields.length >= VIEW_FIELD_LIMIT}
+              />
+            </Tooltip.Trigger>
+            <Tooltip.Portal>
+              <Tooltip.Content classNames='z-[12]'>
+                {view.fields.length >= VIEW_FIELD_LIMIT ? t('field limit reached') : t('add field')} <Tooltip.Arrow />
+              </Tooltip.Content>
```

Comment:
Why should the `label` be different from the tooltip? I’d assume they should align. If you agree, should `IconButton` simply always render a tooltip?

Reply:
i don't like the tooltips but yes we shouldn't require the developer to do this.

Reply:
*always render a tooltip _when it sets `sr-only` on its in-flow label_?

Reply:
the tooltips are not great, but I think not having them is even worse because then you're just left guessing what the icon means. We should work on the tooltips rather than removing them

Reply:
> We should work on the tooltips rather than removing them

Yes, please let’s not leave icon-only buttons totally unlabeled. If tooltips are a blocker, let’s prioritize getting that right rather than giving end-users headaches. We have a path forward with `virtualRef` poppers.

Reply:
Agreed, but in this case the limit should be displayed as a caption inside the form.